nussknacker-dist/src/universal/conf/application.conf

```diff
@@ -40,3 +40,12 @@ countsSettings {
   influxUrl: ${?COUNTS_URL}
   database: "esp"
 }
+
+kafka {
+  kafkaAddress: ${?KAFKA_ADDRESS}
+  kafkaProperties {
+    "schema.registry.url": ${?SCHEMA_REGISTRY_URL}
+  }
+  consumerGroupNamingStrategy: processId-nodeId
+  avroKryoGenericRecordSchemaIdSerialization: true
```

Comment:
Why we have two properties: avroKryoGenericRecordSchemaIdSerialization and autoRegisterRecordSchemaIdSerialization? And why it is here if default is true?

Reply:
i've change default if config not provided